Assets/MRTK/Examples/Demos/HandTracking/Scripts/LeapMotionOrientationDisplay.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.LeapMotion.Input;
+using Microsoft.MixedReality.Toolkit.Input;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Examples
+{
+    /// <summary>
+    /// Returns the orientation of Leap Motion Controller
+    /// </summary>
+    public class LeapMotionOrientationDisplay : MonoBehaviour
+    {
+        [SerializeField] TextMeshProUGUI orientationText;
+        LeapMotionDeviceManagerProfile managerProfile;
+
+        private void Start()
+        {
+            if (GetLeapManager())
+            {
+                orientationText.text = "Orientation: " + GetLeapManager().LeapControllerOrientation.ToString();
+            }
+            else
+            {
+                orientationText.text = "Orientation: Unavailable";
+            }
+        }
+
+        private LeapMotionDeviceManagerProfile GetLeapManager()
+        {
+            if (!MixedRealityToolkit.Instance.ActiveProfile)
+            {
+                return null;
+            }
+            foreach (MixedRealityInputDataProviderConfiguration config in MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.DataProviderConfigurations)
+            {
+                if (config.ComponentType == typeof(LeapMotionDeviceManager))
+                {
+                    managerProfile = (LeapMotionDeviceManagerProfile)config.Profile;
+                    return managerProfile;
+                }
+            }
+            return null;
+        }
+
+    }
```

Comment:
Since this class depends on whether the Leap Unity Modules/ Leap Motion Core package is in the project, we need to add the `LEAPMOTION_CORE_PRESENT` definition around this class.  This definition ensures that this code is not included in a build if the leap motion unity modules are not in the project. 

This is important since I got build errors from this script when I switched the platform to UWP and built. 

To test if the issues have been resolved:
1. If the leap unity modules are in the project, first Separate the Leap Unity Modules from MRTK 
![image](https://user-images.githubusercontent.com/53493796/116938848-37cc5100-ac20-11eb-9a13-7311c91087e1.png)

2. Close Unity and delete the leap Plugins folder from file explorer
3. Reopen Unity 
4. Navigate to the Hand Interaction Example scene 
5. Ctrl + Shift + B for the build settings
6. Switch to UWP
7. Build
8. Check for build errors  

We need to add this guard around the class like this to prevent the build errors:
![image](https://user-images.githubusercontent.com/53493796/116939875-ec1aa700-ac21-11eb-8bcb-d8aec349cb77.png)